Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN git clone https://github.com/akesterson/shunit.git /tmp/shunit \
 WORKDIR /gopath/src/github.com/Azure/acs-engine
 
 # Cache vendor layer
-ADD Makefile test.mk versioning.mk Gopkg.toml Gopkg.lock /gopath/src/github.com/Azure/acs-engine/
+ADD Makefile test.mk versioning.mk Gopkg.toml Gopkg.lock . /gopath/src/github.com/Azure/acs-engine/
```

Comment:
Let's add packer.mk instead of the period. Adding the entire directory along with specific files seems inconsistent.

Reply:
I agree. I think it would be an even better idea to add the entire directory, which we do anyway,  instead of individual files. 
Just adding `packer.mk` to the list of files logs messages like
```
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
```